hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/CryptoTransferSimpleFeesSuite.java

```diff
@@ -0,0 +1,695 @@
+                validateChargedUsdWithin("fungibleSingleTxn", 0.0001, 5.0));
+    }
+
+    @LeakyHapiTest(overrides = {"fees.simpleFeesEnabled"})
+    @DisplayName("TOKEN_FUNGIBLE_COMMON: Multiple fungible tokens")
+    final Stream<DynamicTest> fungibleCommonMultipleTokens() {
+        return runBeforeAfter(
+                cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                cryptoCreate(RECEIVER).balance(0L),
+                tokenCreate(FUNGIBLE_TOKEN)
+                        .tokenType(FUNGIBLE_COMMON)
+                        .initialSupply(1000L)
+                        .treasury(PAYER)
+                        .payingWith(PAYER),
+                tokenCreate(FUNGIBLE_TOKEN_2)
+                        .tokenType(FUNGIBLE_COMMON)
+                        .initialSupply(1000L)
+                        .treasury(PAYER)
+                        .payingWith(PAYER),
+                tokenAssociate(RECEIVER, FUNGIBLE_TOKEN, FUNGIBLE_TOKEN_2),
+                cryptoTransfer(
+                                moving(100, FUNGIBLE_TOKEN).between(PAYER, RECEIVER),
+                                moving(50, FUNGIBLE_TOKEN_2).between(PAYER, RECEIVER))
+                        .payingWith(PAYER)
+                        .signedBy(PAYER)
+                        .fee(ONE_HBAR)
+                        .via("fungibleMultiTxn"),
+                // Expected: $0.0002 (base $0.0001 + 1 extra token type @ $0.0001)
+                // 2 unique token types: first included, second charged
+                validateChargedUsdWithin("fungibleMultiTxn", 0.0002, 5.0));
+    }
+
+    @LeakyHapiTest(overrides = {"fees.simpleFeesEnabled"})
+    @DisplayName("TOKEN_FUNGIBLE_COMMON: Fungible + HBAR mixed transfer")
+    final Stream<DynamicTest> fungibleCommonWithHbar() {
+        return runBeforeAfter(
+                cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                cryptoCreate(RECEIVER).balance(0L),
+                tokenCreate(FUNGIBLE_TOKEN)
+                        .tokenType(FUNGIBLE_COMMON)
+                        .initialSupply(1000L)
+                        .treasury(PAYER)
+                        .payingWith(PAYER),
+                tokenAssociate(RECEIVER, FUNGIBLE_TOKEN),
+                cryptoTransfer(
+                                movingHbar(ONE_HBAR).between(PAYER, RECEIVER),
+                                moving(100, FUNGIBLE_TOKEN).between(PAYER, RECEIVER))
+                        .payingWith(PAYER)
+                        .signedBy(PAYER)
+                        .fee(ONE_HBAR)
+                        .via("fungibleHbarTxn"),
+                // Expected: $0.0001 (same as HBAR-only, no token-specific premium)
+                // Uses 1 signature (PAYER) matching fee schedule includedCount: 1
+                validateChargedUsdWithin("fungibleHbarTxn", 0.0001, 5.0));
+    }
+
+    @LeakyHapiTest(overrides = {"fees.simpleFeesEnabled"})
+    @DisplayName("TOKEN_NON_FUNGIBLE_UNIQUE: Single NFT transfer")
```

Comment:
Disable all of the NFT tests until we have a resolution on which approach we will use for custom fees & nfts in the fee schedule.